data.sql

```diff
@@ -22,4 +22,8 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WH
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom' ; 
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') WHERE name = 'Angemon' OR name = 'Boarmon' ; 
 
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Vet William Tatcher', 45, '2000-04-23'),('Vet Maisy Smith', 26, '2019-01-17'), ('Vet Stephanie Mendez', 64, '1981-05-04'), ('Vet Jack Harkness', 38, '2008-06-08');
+INSERT INTO specializations VALUES (1,1), (1,3), (2,3), (2,4);
```

Comment:
- [ ] Please add field names after `specialization` and before `VALUES` because if you don't mention the field name PostgreSQL will not understand which value is for which field so kindly fix this issue :slightly_smiling_face: 

```suggestion
INSERT INTO specializations(species_id, vet_id) VALUES (1,1), (1,3), (2,3), (2,4);
```